osidb/models.py

```diff
@@ -1147,6 +1147,34 @@ def _validate_exceptional_affectedness_resolution(self):
                 "a exceptional state having no affectedness.",
             )
 
+    def _validate_notaffected_open_tracker(self):
+        """
+        Check whether notaffected products have open trackers.
+        """
+        if (
+            self.affectedness == Affect.AffectAffectedness.NOTAFFECTED
+            and self.trackers.exclude(status="CLOSED").exists()
+        ):
+            self.alert(
+                "flaw_open_tracker_notaffected",
+                f"Affect for {self.ps_module}/{self.ps_component} is marked as "
+                "NOTAFFECTED but has open tracker(s).",
+            )
+
+    def _validate_wontfix_open_tracker(self):
+        """
+        Check whether wontfix affects have open trackers.
+        """
+        if (
+            self.resolution == Affect.AffectResolution.WONTFIX
+            and self.trackers.exclude(status="CLOSED").exists()
+        ):
+            self.alert(
+                "flaw_open_tracker_wontfix",
+                f"Affect for {self.ps_module}/{self.ps_component} is marked as "
+                "WONTFIX but has open tracker(s).",
+            )
+
```

Comment:
Could we document why they're only alerts? I would expect someone changing an affect to `NOTAFFECTED` via the API to get an error if there are any open trackers, do we not want that? and if so, why?